### ISSUE TITLE
Screen reader doesn't read TreeItems properly

### ISSUE THREAD

**Issue by Caitlin-Hennessy** [microsoft/vscode#74079]
In the Live Share panel, we have a Help tab (vscode.TreeDataProvider) containing three menu items (vscode.TreeItems). When an item is selected, NVDA will read the text of the previous item before reading the current item. E.g., when you tab to "Report a problem", NVDA will say "Tweet us your feedback, report a problem".

**Comment by Caitlin-Hennessy** [comment]
![Capture](https://user-images.githubusercontent.com/45697474/58120796-d0e03980-7bba-11e9-8971-e1d1fdf65e05.PNG)

**Comment by joaomoreno** [comment]
@isidorn Can you double check?

**Comment by isidorn** [comment]
Yes I can reproduce this on win with NVDA.
I can reproduce this for every custom tree view I tried.
And can not reproduce this for our builtin custom views. So there must be something special that the custom tree view is doing - it is using the old tree. Thus assigning also to @sandy081 
Do we plan to adopt the new tree for the custom views as well?

**Comment by sandy081** [comment]
@isidorn Yes adopting new tree for custom view is on deck

**Comment by isidorn** [comment]
We will adopt the new tree. Once that is onde this will be fixed.
Thus assigning to @alexr00 so she can close once the new trees are adopted. Thanks

**Comment by alexr00** [comment]
The new tree is in, and I have confirmed that the previous item is not read when tabbing to tree items.

**Comment by isidorn** [comment]
Fancy 🎉
